### PR TITLE
Changes in Fan  schema to accommodate MEX FRU 

### PR DESCRIPTION
**Pull request by lkammath**
This PR has 3 latest commits 

https://gerrit.openbmc-project.xyz/c/openbmc/bmcweb/+/43325
https://gerrit.openbmc-project.xyz/c/openbmc/bmcweb/+/41086
https://gerrit.openbmc-project.xyz/c/openbmc/bmcweb/+/42210

1) Latest updates to Use GetSubTree method instead of mapper service to get the service which is hosting MEX Vs CEC chassis
2) Fan Schema updates for MEX chassis
3) LocationIndicatorActive changes to accommodate MEX Fan

### CODE REVIEW

File: scripts/update_schemas.py [hunk 1]
```diff
@@ -11,6 +11,105 @@
 
 VERSION = "DSP8010_2021.2"
 
+# To use a new schema, add to list and rerun tool
+include_list = [
```
Comment (lxwinspur) [inline scripts/update_schemas.py:15]: Remove these

File: static/redfish/v1/JsonSchemas/Fan/Fan.json [hunk 2]
```diff
@@ -114,7 +114,6 @@
                 "Manufacturer": {
                     "description": "The manufacturer of this fan.",
                     "longDescription": "This property shall contain the name of the organization responsible for producing the fan.  This organization may be the entity from whom the fan is purchased, but this is not necessarily true.",
-                    "readonly": true,
```
Comment (gtmills) [inline static/redfish/v1/JsonSchemas/Fan/Fan.json:117]: Why are we changing this..? From https://redfish.dmtf.org/schemas/Fan.v1_1_0.json

```
              "Manufacturer": {
                    "description": "The manufacturer of this fan.",
                    "longDescription": "This property shall contain the name of the organization responsible for producing the fan.  This organization may be the entity from whom the fan is purchased, but this is not necessarily true.",
                    "readonly": true,
                    "type": [
                        "string",
                        "null"
                    ]
                },
```

Comment (lkammath) [reply static/redfish/v1/JsonSchemas/Fan/Fan.json]: while fixing the conflicts i think i have accidentally deleted it. I have restored it back

**Review by gtmills — APPROVED** [review]

File: redfish-core/lib/fan.hpp [hunk 3]
```diff
@@ -291,147 +300,215 @@ inline void
...
+            "xyz.openbmc_project.ObjectMapper",
+            "/xyz/openbmc_project/object_mapper",
+            "xyz.openbmc_project.ObjectMapper", "GetSubTreePaths",
+            "/xyz/openbmc_project/sensors", 0, sensorInterfaces);
+    }
 }
 
 inline void getFanSpeed(const std::shared_ptr<bmcweb::AsyncResp>& asyncResp,
-                        const std::string& chassisId, const std::string& fanId)
+                       const std::string& chassisId, const std::string& fanId,
```
Comment (gtmills) [inline redfish-core/lib/fan.hpp:396]: This breaks clang-format

**Comment by gtmills** [comment]
jenkins run tests please

**Pull request merged**